corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py

```diff
@@ -235,5 +254,19 @@ def get_correct_xmlns(xform_instance):
     previously_fixed_forms_in_build = get_previously_fixed_forms(build)
     if len(previously_fixed_forms_in_build) == 1:
         return previously_fixed_forms_in_build[0].xmlns
+    elif len(previously_fixed_forms_in_build) == 0:
+        if get_forms_without_xmlns(build):
+            # We don't expect this to ever happen
+            raise BuildHasFormsWithUndefinedXmlns()
+        else:
+            matching_forms = find_matching_forms_by_name(xform_instance, build)
+            if len(matching_forms) == 1:
+                return matching_forms[0].xmlns
+            else:
+                raise CantMatchAForm()
```

Comment:
Should this exception give any more info like there are multiple forms, or no forms?